dev/httpbin/docker-compose.yaml

```diff
@@ -0,0 +1,5 @@
+services:
+  httpbin:
```

Comment:
is this used in anywhere? i think i didn't find it. If it is used, maybe need to add it to readme?

Reply:
Everything in the `dev` directory is not used in the code. It is only for development. I have described how to use it in the PR description. I will add the explaination in the readme. Thanks for the feedback!

Reply:
HTTPBin explaination has been added in the readme.md file.